### DISCUSSION TITLE
Asterisk (*) key listener

### DISCUSSION THREAD

**Discussion by ayozebarrera** [JohannesKlauss/react-hotkeys-hook#907] [Q&A]
Anyone knows how can I detect the asterisk key press ? If I use "*" it will be detect any key

Thank you!

**Comment by JohannesKlauss** [comment]
You can check the actual pressed key in the event that is passed to the callback.

So something like this should work:

```ts
useHotkeys("*", (e) => console.log("has * been pressed?", e.key === '*'));
```
